### PR TITLE
release aplus-v2406.4

### PR DESCRIPTION
**Pull request by m4ra**
**Tasks**
- [ ] PR name contains story or task reference
- [ ] Documentation (docs and inline)
- [ ] Tests (including n+1 and django_assert_num_queries where applicable)
- [ ] Changelog

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -17,6 +19,8 @@ This project (not yet) adheres to [Semantic Versioning](https://semver.org/spec/
 ### Changed
 
 - make wording for ai report toggle more specific
+- added new modifier classes to `PollResults.jsx`, `PollOpenQuestion.jsx` and `PollQuestion.jsx` for customized paddings
+- replaced `<span></span>` with react fragment to keep consistency
```
Comment (m4ra) [inline CHANGELOG.md:23]: @hom3mad3 @goapunk dunno why this changelog wasn't there when I created the 2406.3 release, or I totally missed it, though i used the parser script.

**Review by goapunk — APPROVED** [review]
LGTM (feel free to merge), just out of curiosity:  why have an empty release?

**Comment by m4ra** [comment]
@goapunk how else then? the last commit points aplus v2406.3

**Comment by goapunk** [comment]
true, but as nothing changed I feel like there's no need for a release, I'm not opposing it btw just wondered. If you think having one makes more sense I'm happy to go with it :)

**Pull request closed without merging**